libs/sdk-model/src/execution/measure/index.ts

```diff
@@ -345,6 +347,32 @@ export function isArithmeticMeasure(obj: unknown): obj is IMeasure<IArithmeticMe
     return isMeasure(obj) && isArithmeticMeasureDefinition(obj.measure.definition);
 }
 
+/**
+ * Creates a virtual arithmetic local ID
+ * by appending the virtual arithmetic measure identifier suffix to the given local ID.
+ *
+ * @param localId - The original local ID to which the suffix will be appended.
+ * @returns The new virtual arithmetic local ID with the suffix.
+ */
+export function createVirtualArithmeticLocalId(localId: string): string {
+    return `${localId}${VIRTUAL_ARITHMETIC_MEASURE_IDENTIFIER_SUFFIX}`;
+}
+
+/**
+ * Type guard for checking whether an object is a virtual arithmetic measure based on its local ID.
+ *
+ * @internal
+ *
+ * @param obj - The object to be checked for being a virtual arithmetic measure.
+ * @returns Returns true if the object is a virtual arithmetic measure, false otherwise.
+ */
+export function isVirtualArithmeticMeasure(obj: unknown): obj is IMeasure<IArithmeticMeasureDefinition> {
+    return (
+        isArithmeticMeasure(obj) &&
+        obj.measure.localIdentifier.endsWith(VIRTUAL_ARITHMETIC_MEASURE_IDENTIFIER_SUFFIX)
```

Comment:
Is there some other way how we can mark the measure as virtual? I don't like messing with local identifier. IMHO it is not stable. Local identifier is supposed to be arbitrary ID that is used inside of the insight or execution. If I remember correctly, there are even functions in the code and in e2e tests and Wiremock recordings that do some sort of local identifier normalization, when local IDs are taken and replaced with different values, for example "m1", "m2", etc. You could loose your vital information and the headline will break.

Maybe you could use some sort of map, where you would store virtual metrics IDs send to execution and then when it is returned, map it back to objects or something else you can think of.

Reply:
Updated to introduce a new property to mark the arithmetic measure as virtual, but I see that the arithmetic measure interface is public and I am not sure whether it is acceptable while the user doesn't care what is the virtual arithmetic measure. Please help me to take a look at it!